hera_sim/interpolators.py

```diff
@@ -360,3 +361,33 @@ def __init__(self, datafile, **interp_kwargs):
         super().__init__(datafile, **interp_kwargs)
         self._obj = "bandpass"
         self._check_format()
+
+
+class Reflection(FreqInterpolator):
+    """Complex reflection coefficient interpolator."""
+
+    def __init__(self, datafile, **interp_kwargs):
+        if "interpolator" not in interp_kwargs:
+            interp_kwargs["interpolator"] = "interp1d"
```

Comment:
not sure what the point of this is -- you have hardcoded `interp1d` as the interpolator in the functions, so there's no point in setting `interpolator`?

Reply:
Oh, the code was breaking before this. The issue is that when the class is initialized, it tries to read the file assuming that it contains polynomial coefficients unless the `interpolator` is set to something else. So this is effectively setting the default `interpolator` to be `interp1d` unless specified otherwise. It's hacky for sure, but it's a consequence of me not having enough foresight when I initially wrote this module.

Reply:
Ahhh I see. No problem then.